src/app/loadout/loadout-ui/FashionMods.tsx

```diff
@@ -44,10 +47,18 @@ export function FashionMods({
         className={clsx({ [styles.missingItem]: !canSlotShader })}
         plug={(shaderItem ?? defaultShader) as PluggableInventoryItemDefinition}
       />
-      <PlugDef
-        className={clsx({ [styles.missingItem]: !canSlotOrnament })}
-        plug={(ornamentItem ?? defaultOrnament) as PluggableInventoryItemDefinition}
-      />
+      {ornamentItem ? (
+        <PlugDef
+          className={clsx({ [styles.missingItem]: !canSlotOrnament })}
+          plug={ornamentItem as PluggableInventoryItemDefinition}
+        />
+      ) : (
+        <PressTip tooltip={<div>{t('FashionDrawer.NoPreference')}</div>}>
+          <div className={clsx('item', styles.missingItem)}>
+            <DefItemIcon itemDef={defaultOrnament} />
+          </div>
+        </PressTip>
```

Comment:
Indicates in the main views whether we are forcibly changing to default or not by more than the plug opacity

<img width="433" alt="Screen Shot 2022-09-09 at 9 42 19 pm" src="https://user-images.githubusercontent.com/7344652/189346367-a6e25499-2773-455d-8458-58fbefd5117a.png">

Reply:
I would like to have this be more complete and indicate whether there exists an ornament at all but you need the socket for that and I don't think it is worth the effort to plumb it all up.